bolt-service/test/test.sh

```diff
@@ -0,0 +1 @@
+curl --insecure -X POST -H "Content-Type: application/json" -d @request.json -E ../certs/127.0.0.1.crt --key ../certs/127.0.0.1.key -k https://localhost:8144/ssh/run_task
```

Comment:
I don't think you need the `--insecure` here, and might try sending the CA as well instead of `-k`